addons/website/views/snippets/s_accordion.xml

```diff
@@ -0,0 +1,61 @@
+                    </div>
+                </div>
+            </div>
+            <div class="accordion-item position-relative z-1" data-name="Item" role="presentation">
+                <a href="#" class="accordion-header accordion-button collapsed justify-content-between gap-2 bg-transparent h6-fs fw-bold text-decoration-none text-reset shadow-none transition-none" t-attf-data-bs-target="#myCollapseTab{{uniq}}_2" role="tab" t-attf-aria-controls="myCollapseTab{{uniq}}_2" data-bs-toggle="collapse" aria-expanded="false"><span class="flex-grow-1">Links to other Websites</span></a>
+                <div t-attf-id="myCollapseTab{{uniq}}_2" class="accordion-collapse collapse" data-bs-parent="#myCollapse" role="tabpanel">
+                    <div class="accordion-body">
+                        <p>Although this Website may be linked to other websites, we are not, directly or indirectly, implying any approval, association, sponsorship, endorsement, or affiliation with any linked website, unless specifically stated herein.</p>
+                        <p>You should carefully review the legal statements and other conditions of use of any website which you access through a link from this Website. Your linking to any other off-site pages or other websites is at your own risk.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="accordion-item position-relative z-1" data-name="Item" role="presentation">
+                <a href="#" class="accordion-header accordion-button collapsed justify-content-between gap-2 bg-transparent h6-fs fw-bold text-decoration-none text-reset shadow-none transition-none" t-attf-data-bs-target="#myCollapseTab{{uniq}}_3" role="tab" t-attf-aria-controls="myCollapseTab{{uniq}}_3" data-bs-toggle="collapse" aria-expanded="false"><span class="flex-grow-1">Use of Cookies</span></a>
+                <div t-attf-id="myCollapseTab{{uniq}}_3" class="accordion-collapse collapse" data-bs-parent="#myCollapse" role="tabpanel">
+                    <div class="accordion-body">
+                        <p>Website may use cookies to personalize and facilitate maximum navigation of the User by this site. The User may configure his / her browser to notify and reject the installation of the cookies sent by us.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<template id="s_accordion_options" inherit_id="website.snippet_options">
+    <xpath expr="." position="inside">
+        <div data-selector=".s_accordion" data-target=".accordion">
+            <we-select string="Card Style">
+                <we-button title="Default" data-select-class="s_accordion_default">Default</we-button>
+                <we-button title="Borderless" data-select-class="accordion-flush">Borderless</we-button>
+                <we-button title="Boxed" data-select-class="s_accordion_boxed">Boxed</we-button>
```

Comment:
I'm not sure about the UX, what about:

- Reword "_Default_" to "_Boxed_": default  it's not really explicative for the user while boxed it's closer to the final visual result
- Reword the "_Borderless_" to "_Side borders_": transform it to a switch, active by default and visible when the style is   "_Boxed_" (ex default) only.
- Reword "_Boxed_" to "_Highlight Active_":  it's what the option does at the end. Active elements become more predominant thanks to their background

<img width="262" alt="image" src="https://github.com/user-attachments/assets/694ae56a-c1c7-4a3b-965b-6ab1f20f2034">